### ISSUE TITLE
Screenshot does not resets it's dimensions after other screenshot / scrollIntoView

### ISSUE THREAD

**Issue by sasf54** [cypress-io/cypress#4789]
### Current behavior:
I'm trying to take a couple of screenshots of elements (not full screen), and I think the old parameters are stuck, or just added to.

### Desired behavior:

Get a screenshot of all 3 element, with right X,Y top right corner, and right width and height

- `01_transversal1.png` is fine (right position + dimensions)
- `02_actions1.png`  is failed (wrong position (transversal) + right dimensions)
- `03_transversal1.png` is failed (wrong position (actions) + right dimensions)

### Steps to reproduce: (app code and test code)

```js
it('screenshots', () => {
  cy.visit('https://example.cypress.io')
  cy.get('.home-list>li').eq(1).screenshot('01_transversal1')
  cy.get('.home-list>li').eq(2).screenshot('02_actions1')
  cy.get('.home-list>li').eq(1).screenshot('03_transversal1')
})

```

### Versions

Docker runner:
FROM cypress/included:3.4.0
Cypress:    3.4.0                                                                              Electron 61 (headless) 

`01_traversal1.png`

![01_transversal1](https://user-images.githubusercontent.com/266505/61715883-54a2d780-ad5e-11e9-9ad5-4e8d028ae62f.png)

`02_actions1.png`

![02_actions1](https://user-images.githubusercontent.com/266505/61715885-553b6e00-ad5e-11e9-84c2-fa3da07a2227.png)

`03_transversal1`

![03_transversal1](https://user-images.githubusercontent.com/266505/61715886-553b6e00-ad5e-11e9-9e78-87e48460ff04.png)

**Comment by sasf54** [comment]
Some other information about the issue:
If I run it natively (newest 3.4.0) with built in electron,
it works fine.
For a workaround:
add a scrollIntoView() and a wait as below:

    cy.get(".home-list>li").eq(1).scrollIntoView()
    cy.wait(100)
    cy.get(".home-list>li").eq(1).screenshot("01_transversal1")
    cy.get(".home-list>li").eq(2).scrollIntoView()
    cy.wait(100)
    cy.get(".home-list>li").eq(2).screenshot("02_actions1")
    cy.get(".home-list>li").eq(1).scrollIntoView()
    cy.wait(100)
    cy.get(".home-list>li").eq(1).screenshot("03_transversal1")

**Comment by jennifer-shehane** [comment]
I'm actually not experiencing this same issue. 

- What version of Chrome are you using when you run these?
- What OS?
- Anthing in particular with screen dimensions?

`01_traversal1.png`

![01_transversal1](https://user-images.githubusercontent.com/1271364/62135842-f13e1a00-b308-11e9-9025-9f080b826076.png)

`02_actions1`

![02_actions1](https://user-images.githubusercontent.com/1271364/62135864-0024cc80-b309-11e9-8512-6a7e81abf7ef.png)

`03_transversal1.png`

<img width="641" alt="Screen Shot 2019-07-30 at 4 59 59 PM" src="https://user-images.githubusercontent.com/1271364/62135897-0dda5200-b309-11e9-9887-5f805ef32e74.png">

**Comment by sasf54** [comment]
Hi Jennifer,

As I mentioned, it's a docker / docker worker issue.
Docker runner:
FROM cypress/included:3.4.0
Cypress: 3.4.0 Electron 61 (headless)

Os: Xubuntu 18.04 -> docker -> cypress/included:3.4.0 -> Electron 61 (headless) -> issue
os: Xubuntu 18.04 -> Cypress 3.4.0 -> Electron 61 -> no issue

Screen dimensions:
I don't think so. I executed the same test on both.
(I suspect it's not about the screen resolution, because the screenshot is from an element, and the screen (viewport) size is fine, only the position is wrong)

**Comment by jennifer-shehane** [comment]
Since this issue hasn't had activity in a while, we'll close the issue until we can confirm this is still happening. Please comment if there is new information to provide concerning the original issue and we'd be happy to reopen.
